cli/internal/prune/prune.go

```diff
@@ -65,11 +64,6 @@ func parsePruneArgs(args []string) (*PruneOptions, error) {
 		return nil, errors.Errorf("At least one target must be specified.")
 	}
 
-	cwd, err := os.Getwd()
```

Comment:
This one is more of a judgment call vs `run`, but passing `cwd` in here and keeping it on the options struct will result in a lot less churn.

Reply:
Updated.